addons/website/controllers/main.py

```diff
@@ -336,7 +336,17 @@ def website_configurator(self, step=1, **kwargs):
 
     @http.route(['/website/social/<string:social>'], type='http', auth="public", website=True, sitemap=False)
     def social(self, social, **kwargs):
-        url = getattr(request.website, 'social_%s' % social, False)
+        default_values = {
+            'facebook': 'https://www.facebook.com/Odoo',
+            'github': 'https://github.com/odoo',
+            'linkedin': 'https://www.linkedin.com/company/odoo',
+            'youtube': 'https://www.youtube.com/user/OpenERPonline',
+            'instagram': 'https://www.instagram.com/odoo.official',
+            'twitter': 'https://twitter.com/Odoo',
+            'tiktok': 'https://www.tiktok.com/@odoo',
+        }
```

Comment:
Don't we want to make this list extensible ? E.g. if someone develops a `website_social_media_twitch` to further populate the `s_social_media` snippet with a default `https://www.twitch.tv/odoo` ?

Reply:
Indeed, doesn't cost much making it extensible, it's done